content/docs/get-started/aws/modify-program.md

```diff
@@ -18,44 +18,51 @@ Now that your S3 bucket is provisioned, let's add an object to it. First, create
 $ mkdir site
 ```
 
-Next create an `index.html` file you will upload to your bucket.
+Next, create a new `index.html` file with some content you will upload to your bucket.
```

Comment:
Not your PR, but `with some content you will upload` is a bit awkard. Suggest maybe:

```suggestion
Next, create a new `index.html` file containing a bit of content.
```

.. or the like.